src/components/Navbar/styledComponents.tsx

```diff
@@ -0,0 +1,98 @@
+import styled from "styled-components";
+import { minDevice } from "../../assets/styles/breakpoints";
+import { activeLink } from "../../assets/styles/colors";
+import P from "../P";
+
+const NavbarDiv = styled.div`
+  height: 5em;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 1.16em 0 0.81em;
+
+  &.active {
+    box-shadow: 0px 1px 10px rgba(0, 0, 0, 0.08);
+  }
+
+  @media ${minDevice.desktopTablet} {
+    margin: 0 1.5em 0 1.31em;
+    height: 4em;
+    box-shadow: none;
+  }
+
+  @media ${minDevice.desktopL} {
+    margin: 0 6.25em 0 6.06em;
+    height: 4.5em;
+  }
+
+  @media ${minDevice.desktopXL} {
+    margin: 0 12.31em 0 12.5em;
+  }
+`;
+const NavbarMenu = styled.div`
```

Comment:
We can improve this navigation when it comes to html semantics. ;)
You can taka look at this piece of docs: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav
I'd go with `<nav> <ol> and <li>`, you don't need links for now, as we'll use Router. ;)

Reply:
Also a tiny read around it: https://dev.to/ameseee/semantic-html-how-can-you-tell-13am

Reply:
Thanks, I've made that changes :)